hazelcast/src/test/java/com/hazelcast/map/TestBackupUtils.java

```diff
@@ -0,0 +1,192 @@
+package com.hazelcast.map;
+
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.instance.Node;
+import com.hazelcast.internal.partition.InternalPartition;
+import com.hazelcast.internal.partition.InternalPartitionService;
+import com.hazelcast.internal.serialization.InternalSerializationService;
+import com.hazelcast.map.impl.MapService;
+import com.hazelcast.map.impl.MapServiceContext;
+import com.hazelcast.map.impl.PartitionContainer;
+import com.hazelcast.map.impl.recordstore.RecordStore;
+import com.hazelcast.nio.Address;
+import com.hazelcast.nio.serialization.Data;
+import com.hazelcast.spi.impl.NodeEngineImpl;
+import com.hazelcast.spi.partition.IPartition;
+import com.hazelcast.test.AssertTask;
+import com.hazelcast.test.HazelcastTestSupport;
+
+import static com.hazelcast.test.HazelcastTestSupport.assertTrueEventually;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+/**
+ * Convenience for accessing backup records.
+ *
+ */
+public final class TestBackupUtils {
+    private static final int DEFAULT_REPLICA_INDEX = 1;
+
+    private TestBackupUtils() {
+
+    }
+
+    /**
+     * Access backup records in a given replica. Use {@link #newMapAccessor(HazelcastInstance[], String)}
+     * to create an instance for your data structure.
+     *
+     * @param <K>
```

Comment:
The `@param` and `@return` JavaDoc in this class should be completed or removed, in this state there will be a lot of warnings in IDEA.